### DISCUSSION TITLE
Are auto merges supported?

### DISCUSSION THREAD

**Discussion by AquilesGomez** [pwntester/octo.nvim#497] [General]
I have a repository that takes a long time to build. GitHub supports the option to enable auto-merges once all requirements have been met. In the `gh` cli, it's fairly easy to pull this off via:
`gh pr merge --auto`.

I tried doing something similar via:
`Octo pr merge auto` and `Octo pr merge --auto`, but neither worked. Am I using the wrong syntax or is that feature not supported?

**Comment by williambdean** [comment]
Hi @AquilesGomez,

It is not currently supported. Are you interested in making a PR? 

I've layout some links in the issue here: #917
